packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -792,7 +790,7 @@ function runActTests(label, render, unmount, rerender) {
       }
     });
     describe('warn in prod mode', () => {
-      it('warns if you try to use act() in prod mode', () => {
+      xit('warns if you try to use act() in prod mode', () => {
```

Comment:
I could probably remove this test. Or rewrite it to run across the dev/prod+testing matrix.